### PR TITLE
Introduce mybinder

### PR DESCRIPTION
**Pull request by BjoernLudwigPTB**
Introduce all necessary changes, to allow for execution of our notebooks on [mybinder.org](https://mybinder.org/v2/gh/PTB-PSt1/PyDynamic_tutorials/introduce_mybinder
).

Please click on _Files changed_ on the top of this page, check the applied changes and let me know via the _comment_ feature, what you think about wording and the rest.

If any questions arise, let's chat!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,7 +4,18 @@ This is a collection of tutorials and examples to document, explain and illustra
...
+ ### Quick start in current browser session
+ 
+ To start working in the notebooks directly in the browser, click 
+ [![Binder](https://mybinder.org/badge_logo.svg)](https://mybinder.org/v2/gh/PTB-PSt1/PyDynamic_tutorials/introduce_mybinder).
+ 
+ ### Get a local copy to work offline
+
+To get started on your local machine basically follow these simple steps, which we
+will take you through in
 the following:
```
Comment (anupam-prasad) [inline README.md:19]: I feel "To get started on your local machine, follow these simple steps:" is enough.

**Review by anupam-prasad — COMMENTED** [review]
Minor comment on lines 17 to 19.

File: README.md (hunk 1 above)
Comment (BjoernLudwigPTB) [reply README.md]: Resolve in 3b9d25c93de4c3da941a6ac9a0f66722a2460769

**Comment by BjoernLudwigPTB** [comment]
> Minor comment on lines 17 to 19.

Thanks for the suggestion. Resolved... ready for approval and the merge?

**Review by anupam-prasad — APPROVED** [review]

**Comment by anupam-prasad** [comment]
> > Minor comment on lines 17 to 19.
> 
> Thanks for the suggestion. Resolved... ready for approval and the merge?

Yep. Done.

**Review by anupam-prasad — APPROVED** [review]

**Review by anupam-prasad — APPROVED** [review]

**Review by anupam-prasad — APPROVED** [review]

**Pull request merged**